requirements.txt

```diff
@@ -7,6 +7,7 @@
 
 setuptools
 altgraph
+pyinstaller-hooks-contrib >= 2020.5
```

Comment:
I wonder whether we actually want this. The main repo does not (or at least should not) have any tests for the hooks defined in hooks-contrib. Thus for development we do not need this package. @Legorooj WDYT?

Reply:
@htgoebel This is the main requirements file which is meant to be kept level with `setup.cfg`. I added it as a release dependency in both files, seeing as they both say to keep them mirrored. As for tests, CI won't run any contrib tests; I've configured it not to. 

And we do actually need this package installed for development otherwise the Sphinx tests fail; they're dependent on Jinja2 which had it's hook moved.

Reply:
Okay, so we'll keep them for now. Thanks